CHANGELOG.rst

```diff
@@ -0,0 +1,328 @@
+===========================
+Netbox.Netbox Release Notes
+===========================
+
+.. contents:: Topics
+
+
+v1.0.0
+======
+
+Bugfixes
+--------
+
+- Fix query_dict for device_bay/interface_template to use ``devicetype_id`` (https://github.com/netbox-community/ansible_modules/issues/282)
+- This expands the fix to all `_template` modules to use `devicetype_id` for the query_dict when attempting to resolve the search (https://github.com/netbox-community/ansible_modules/pull/300)
+
+v0.3.1
+======
+
+Bugfixes
+--------
+
+- Default ``validate_certs`` to ``True`` (https://github.com/netbox-community/ansible_modules/issues/273)
+
+v0.3.0
+======
+
+Minor Changes
+-------------
+
+- Add ``local_context_data`` and ``flatten_local_context_data`` option to ``nb_inventory`` (https://github.com/netbox-community/ansible_modules/pull/258)
+- Add ``local_context_data`` option to ``netbox_device`` (https://github.com/netbox-community/ansible_modules/pull/258)
+- Add ``virtual_chassis``, ``vc_position``, ``vc_priority`` to ``netbox_device`` options (https://github.com/netbox-community/ansible_modules/pull/251)
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- To pass in integers via Ansible Jinja filters for a key in ``data`` that requires querying an endpoint is now done by making it a dictionary with an ``id`` key.
+  The previous behavior was to just pass in an integer and it was converted when normalizing the data, but some people may have names that are all integers and those were being converted erroneously so we made the decision to change
+  the method to convert to an integer for the NetBox API.
+  tasks:
+    - name: Create device within NetBox with only required information
+      netbox_device:
+        netbox_url: http://netbox-demo.org:32768
+        netbox_token: 0123456789abcdef0123456789abcdef01234567
+        data:
+          name: Test66
+          device_type:
+            id: "{{ some_jinja_variable }}"
+          device_role: Core Switch
+          site: Test Site
+          status: Staged
+        state: present
```

Comment:
The code block will look a lot better if you use `::` to format it (https://docutils.sourceforge.io/docs/user/rst/quickstart.html#preformatting-code-samples):
```suggestion

  ::

      tasks:
        - name: Create device within NetBox with only required information
          netbox_device:
            netbox_url: http://netbox-demo.org:32768
            netbox_token: 0123456789abcdef0123456789abcdef01234567
            data:
              name: Test66
              device_type:
                id: "{{ some_jinja_variable }}"
              device_role: Core Switch
              site: Test Site
              status: Staged
            state: present
```
You'll need to update the entry in `changelog.yaml` correspondingly and run `antsibull-changelog generate` to update the .rst file.